src/ophys_etl/transforms/data_loaders.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+from collections import namedtuple
+
+motion_border = namedtuple('motion_border', 'left right up down')
+
+
+def get_max_correction_border(motion_correction_path: Path,
```

Comment:
Instead of having this wrapper function around `_get_max_correction_values` why not have a argschema/marshmallow post-load that automatically reads the motion_correction_path into a dataframe? Then you don't even need this wrapper function.

Reply:
Removed the wrapper function and moved the .csv file opening to the convert_rois module